.github/actions/free-disk-space/action.yml

```
@@ -3,8 +3,13 @@ description: 'Frees disk space on the runner'
 runs:
   using: "composite"
   steps:
-    - run: |
+    - name: Print disk space before cleanup
+      run: |
         df -h
+      shell: bash
+    - name: Free Disk Space Linux
+      if: matrix.platform != 'macos-latest'
```

```suggestion
      if: runner.os == 'Linux'
```

Shouldn't this use the `runner` context to decouple from the use of `matrix'?

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#runner-context

---

I don't know, decoupling makes more generic and usable outside a matrix I suppose?